registration/src/main/proto/lobby/registration/conference.proto

```diff
@@ -70,10 +74,12 @@ message ConferenceAccessCode {
     string value = 1;
 }
 
-// The part of the URL used to access the conference.
-message Slug {
-    // The string value of the slug.
-    string value = 1;
+// The publishing status of the conference.
+enum ConferencePublishingStatus {
```

Comment:
Can we publish something, which is not 'conference'? If not, I'd just have PublishingStatus.